includes/functions/metadata.php

```diff
@@ -193,168 +237,210 @@ function resource_data_init() {
 	$prefix = 'lc_resource_';
 
 	$cmb = new_cmb2_box(
-		array(
+		[
 			'id'           => 'resource_data',
 			'title'        => pll__( 'Resource Data' ),
-			'object_types' => array( 'lc_resource' ),
+			'object_types' => [ 'lc_resource' ],
 			'context'      => 'normal',
 			'priority'     => 'high',
 			'show_names'   => true,
-		)
+		]
 	);
 
 	$cmb->add_field(
-		array(
+		[
 			'name'        => pll__( 'Permanent Link' ),
 			'description' => pll__( 'A permanent link to the resource.' ),
 			'id'          => $prefix . 'permanent_link',
 			'type'        => 'text_url',
-			'protocols'   => array( 'http', 'https' ),
+			'protocols'   => [ 'http', 'https' ],
```

Comment:
@jhung Added http protocol support in https://github.com/platform-coop-toolkit/learning-commons-framework/pull/84/commits/4857c36a10dfea441ee2d3ab35604cf81cb3ee31.